testing/smoke-test-utils/src/main/kotlin/net/corda/smoketesting/NodeProcess.kt

```diff
@@ -33,6 +35,7 @@ class NodeProcess(
     companion object {
         const val CORDAPPS_DIR_NAME = "cordapps"
         private val log = contextLogger()
+        private val schemaCreationTimeOutSeconds: Long = 180
```

Comment:
```suggestion
        private const val schemaCreationTimeOutSeconds: Long = 180
```
Total nitpick I know